pkgs/development/python-modules/attrs-strict/default.nix

```
@@ -0,0 +1,38 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  pythonOlder,
+  python3
+}:
+
+buildPythonPackage rec {
+  pname = "attrs_strict";
```

```suggestion
  pname = "attrs-strict";
```

PEP503 normalization